apps/dolly-frontend/src/main/js/src/components/bestillingsveileder/stegVelger/steg/steg1/paneler/Adresse.tsx

```diff
@@ -40,6 +40,9 @@ AdressePanel.heading = 'Adresser'
 
 AdressePanel.initialValues = ({ set, opts, del, has }: any) => {
 	const { identtype } = opts
+	const { identMaster } = opts
```

Comment:
Kan gjerne slå sammen disse to linjene til `const { identtype, identMaster } = opts`

Reply:
Ah, visste jeg ikke, takk! 👍